ui/yarn.lock

```diff
@@ -5657,24 +5643,16 @@ indent-string@^4.0.0:
   resolved "https://registry.yarnpkg.com/indent-string/-/indent-string-4.0.0.tgz#624f8f4497d619b2d9768531d58f4122854d7251"
   integrity sha512-EdDDZu4A2OyIK7Lr/2zG+w5jmbuk1DVBnEwREQvBzspBJkCEbRa8GxU1lghYcaGJCnRWibjDXlq779X1/y5xwg==
 
-inflight@^1.0.4:
```

Comment:
this is the vulnerable package.  It is abandoned, and has no safe version.  https://security.snyk.io/vuln/SNYK-JS-INFLIGHT-6095116

Reply:
The Glob maintainer will not be releasing a non-vulnerable version of Glob 8. https://github.com/isaacs/node-glob/issues/573